src/components/TextEditor.vue

```diff
@@ -422,13 +423,8 @@ https://github.com/ckeditor/ckeditor5/issues/1142
 	padding :4px 8px !important;
 	display : block;
 	width : 100% !important;
- }
- .ck-off{
```

Comment:
` .ck-list__item{
	.ck-off{
	background:var(--color-main-background)!important;
 	}
	 .ck-on {
	background:var(--color-primary-element-light)!important;
 }
 }`

Reply:
This way the change only applies to list items and the toolbar styles are not broken

Reply:
thank you!